config/checkstyle/checkstyle.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <!--
 
-    Copyright 2017 The OpenTracing Authors
+    Copyright 2017-2018 The OpenTracing Authors
```

Comment:
There should not be this change. Only if you edited the file which is not the case